requirements.txt

```
@@ -30,7 +30,7 @@ redis==2.9.1
 requests==2.20.0
 retrying==1.3.3
 six==1.11.0
-SQLAlchemy==0.9.7
+SQLAlchemy==1.3.0
```

Maybe consider a smaller step, before big jump to 1.3
```suggestion
SQLAlchemy==1.1.18
```